message_ix/model/MESSAGE/parameter_def.gms

```diff
@@ -73,6 +73,11 @@ Parameters
 * Parameters of the `Resources` section
 * -------------------------------------
 *
+* The quantity of the resources in |MESSAGEix| is represented by two parameters: ``resource_volume`` and ``resource_remaining``.
+* The first parameter is the volume of resources at the start of the model horizon while the second is the maximum extraction relative
+* to remaining resources by year. The maximum extraction of the resources (by grade) in a year is constrained by the parameter
```

Comment:
I don't understand the sentence: _while the second is the maximum extraction relative to remaining resources by year_ here. Do you think it is enough? Probably I missed other previous descriptions that could help me understand it.

Shouldn't it be that the second are just the remaining sources per year?